vpp/README.md

```diff
@@ -0,0 +1,21 @@
+# VPP Debian/Ubuntu VM
+
+To download a compatible image of the Ubuntu VM execute the [download.sh](download.sh) script that will download a cloud-init image of Ubuntu from <https://ipng.ch/media/vpp-proto/vpp-proto-bookworm-20250607.qcow2.lrz>. The version is set in the script and can be changed manually.
+
```

Comment:
I suggest you use the non-datestamped version, as this one may eventually be cleaned up.
https://ipng.ch/media/vpp-proto/vpp-proto-bookworm.qcow2.lrz

Also, I will soon start shipping Trixie images, and stop shipping Bullseye images. I'll make an update to the images today.